controllers/apiController.js

```diff
@@ -0,0 +1,59 @@
+var Todos = require('../models/todoModels');
+var bodyParser = require('body-parser');
+
+module.exports = function(app) {
+
+    app.get('/api/todo/:username', function(req, res) {
```

Comment:
Need to add jsDocs to these methods for future maintenance and documentation. 

The following issue has been created for that: #3